WorkflowSwiftUI/Sources/Store+Preview.swift

```diff
@@ -12,23 +12,17 @@ public struct StaticStorePreviewContext {
     }
 
     public func makeStateAccessor<State>(state: State) -> StateAccessor<State> {
-        StateAccessor(
-            state: state,
-            sendValue: { _ in }
-        )
+        .static(state: state)
     }
 
     public func makeActionModel<State, Action>(
         state: State
     ) -> ActionModel<State, Action> {
-        ActionModel(
-            accessor: makeStateAccessor(state: state),
-            sendAction: makeSink(of: Action.self).send
-        )
+        .static(state: state)
     }
 }
 
-extension Store {
+public extension Store {
```

Comment:
Please keep the access control on the individual methods.

Reply:
I agree, but the linter did this!

Reply:
approve me! https://github.com/square/workflow-swift/pull/315